torch/nn/modules/loss.py

```diff
@@ -1266,12 +1266,13 @@ class CTCLoss(_Loss):
         - Output: scalar. If :attr:`reduction` is ``'none'``, then
           :math:`(N)`, where :math:`N = \text{batch size}`.
 
-    Example::
+    Example1::
```

Comment:
This isn't just a free-form headline, but a tag from a fixed vocabulary. You want to have `Examples::` here and then use comments to split the examples
Please search for `Examples::` in this file for how it is done for other losses.

Reply:
its my  honor to get this suggestions from your